velox/functions/sparksql/IntervalFunctions.h

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
Perhaps rename this header as `MakeDTInterval`. It is recommended to add a separate file for each function.